### ISSUE TITLE
Pagination for Participant Lists etc

### ISSUE THREAD

**Issue by listerr** [indico/indico#6424]
**Is your feature request related to a problem? Please describe.**
Some of our attendee/participant lists can get pretty long, say, 200 participants.

I can't see a way in Indico we can have pagination (e.g., so it displays 25/50 attendees at a time with next/back buttons)

Would also be useful to have a search or filter option for the attendee list, rather than having to use the browser's search in page feature.

**Describe the solution you'd like**

Pagination for participant lists (ala datatables+bootstrap: https://datatables.net/examples/styling/bootstrap5)

Possibly filter search option

**Comment by ThiefMaster** [comment]
I assume you mean the list in the public part of the event, and not in the management area?

And I'm curious, what's wrong with using the normal search functionality of the web browser if you want to find someone? :)

(not saying this is a bad suggestion, just curious!)

**Comment by listerr** [comment]
> I assume you mean the list in the public part of the event, and not in the management area?

Yes. Although I think the management area could use similar functionality.

> And I'm curious, what's wrong with using the normal search functionality of the web browser if you want to find someone? :)
> 
> (not saying this is a bad suggestion, just curious!)

Nothing inherently wrong with it, but it's useful to have a quick filter, say, if you want to quickly find and display only participants with a particular affiliation.  The way to do that now is to sort by Affiliation and then scroll down until you find them in the list.

I'd also like to have a tabbed Participant List  e.g. where the public attendee list is made up of multiple separate registration forms, have tabs at the top to show each, rather than having to scroll all the way down to the bottom of one to see the other.

e.g.

| In-Person Participants     |   Remote Participants   |
| --------------------------- | ------------------------ | 

|  ... List of attendees for selected tab                         | 
| --------------------------- | 

Instead of:

**In-Person Participants** 

(Long list of participants)

**Remote Participants**

(List of remote participants)

Limiting the number of rows output would make it a bit friendlier to navigate and more embeddable into other layouts or websites, for example.

**Comment by adam-parker1** [comment]
Just adding a +1 to this enhancement

We just got a request for search functionality in the registration management area from some of our event organisers (where we have 800+ participants), so this would be very useful for us also
